CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,135 @@
+
+This statement thanks the following draws directly from Apache Software Foundation [code of conduct](https://www.apache.org/foundation/policies/conduct.html) 
+, which in turn draws for on the following for content and inspiration:
+
+* [CouchDB Project Code of conduct](http://couchdb.apache.org/conduct.html)
+* [Fedora Project Code of Conduct](http://fedoraproject.org/code-of-conduct)
+* [Speak Up! Code of Conduct](http://web.archive.org/web/20141109123859/http://speakup.io/coc.html)
+* [Django Code of Conduct](https://www.djangoproject.com/conduct/)
+* [Debian Code of Conduct](https://www.debian.org/vote/2014/vote_002)
+* [Twitter Open Source Code of Conduct](https://github.com/twitter/code-of-conduct/blob/master/code-of-conduct.md)
+* [Mozilla Code of Conduct/Draft](https://wiki.mozilla.org/Code_of_Conduct/Draft#Conflicts_of_Interest)
+* [Python Diversity Appendix](https://www.python.org/community/diversity/)
+* [Python Mentors Home Page](http://pythonmentors.com)
+
+
+# Slack Community Guidelines
+
+If you decide to join the [Community Slack](https://kroxylicious.slack.com/), please adhere to the following rules:
+
+**1. Treat everyone in the community with respect.**
+
+- We strive to make this community a warm place for people from all industries, use cases, geographies, and backgrounds. Harassment of any kind is not acceptable and won’t be tolerated.
+- Please follow the guidelines as outlined in the Kroxylicious Community [code of conduct here](https://github.com/kroxylicious/kroxylicious/CODE_OF_CONDUCT.md).
+
+**2. Use the right channel.**
+
+- Channels are an effective way to organize and focus discussions while also empowering members to opt-in to the types of content they’re interested in. When questions are posted or discussions are started in the wrong channel, it dilutes the trust of the members in the channel and, more practically, makes it harder for your questions to be answered.
+
+**3. Ask thoughtful questions.**
+
+- We’re all here to help each other out. The best way to get help is by investing effort into your questions. First check and see if your question is answered in the [Kroxylicious documentation](https://kroxylicious.io/kroxylicious/). You can also check GitHub trackers to see if your feature requests or bug report [GitHub issues](https://github.com/kroxylicious/kroxylicious/issues).
+
+If you still feel the need to ask a question, make sure you include:
+
+- The steps you’ve already taken.
+- Relevant details presented cleanly: text stacktraces, formatted markdown, Kroxylicious config yaml and or Kubernetes resources (ideally in yaml). Please don’t paste large blocks of code unformatted or post photos of your screen from your phone.
+- The specific question you have or the specific type of help you're seeking.
```

Comment:
Whilst I  agree with the sentiment of what you say here, I question whether it belongs in the https://github.com/kroxylicious/kroxylicious/blob/main/.github/ISSUE_TEMPLATE/bug_report.md

Reply:
Fair. I slightly think good points are worth repeating :D I'm inclined to leave this as it is and try to find some time to make sure the issue template asks for the same things